### PR TITLE
Update api.md to reflect data needed by design

### PR DESCRIPTION
**Pull request by Brochua**
Updates the developer and project models to allow for the design suggest in about/name and projects/

### CODE REVIEW

**Review by Yan2arb4 — APPROVED** [review]
Looks good!

File: api.md [hunk 1]
```diff
@@ -45,7 +47,7 @@ PROJECT STATUSES = {
 	"O": "Ongoing",
 	"U": "Upcoming",
 	"C": "Completed",
-    "H": "On Hold",
+        "H": "On Hold",
```
Comment (RiChaarani) [inline api.md:50]: You made it say ouch. hahaha

**Review by RiChaarani — APPROVED** [review]
Looks good, i like it

**Review by Sprytte — APPROVED** [review]
Changes look good!

**Pull request merged**